### DISCUSSION TITLE
How to enable livenessProbe and livenessProbe  for PostgREST docker image in k8s

### DISCUSSION THREAD

**Discussion by sirijagadeesh** [PostgREST/postgrest#1891] [Q&A] [answered]
Hi all,

I am using postgREST:v7.0.1 (docker image) to test in k8s. But we need enable livenessProbe and livenessProbe.
How can i enable for PostgREST:v7.0.1 images ???

**Comment by colearendt** [comment, marked as answer]
Related discussion: https://github.com/PostgREST/postgrest/issues/1933

Basically, the idea is to use `HEAD` requests at `/`. Kubernetes does not support this, so the options today are either to use `GET /`, or create your own health check endpoint as an `rpc` in your database.

**Comment by steve-chavez** [comment]
There's now proper health check endpoints on the latest pre-release: https://github.com/PostgREST/postgrest/releases/tag/v9.0.0.20220107
